scripts/upgrade.sh

```diff
@@ -47,7 +47,25 @@ kill_k3s_process() {
     info "Successfully Killed old k3s process $K3S_PID"
 }
 
+check_hash(){
+    sha_cmd="sha256sum"
+
+    if [ ! -x "$(command -v $sha_cmd)" ]; then
+    sha_cmd="shasum -a 256"
+    fi
+
+    if [ -x "$(command -v $sha_cmd)" ]; then
+
+    K3S_RELEASE_CHECKSUM=$(echo $K3S_RELEASE_CHECKSUM | sed "s/sha256sum.txt/sha256sum-amd64.txt/g")
```

Comment:
This is because the lack of `-amd64` suffix on builds for that arch?

Reply:
yes